docs/source/release/v6.6.0/mantidworkbench.rst

```diff
@@ -7,36 +7,77 @@ Mantid Workbench Changes
+- Mantid Workbench is now compatible with matplotlib v3.6
+- Sample shapes can now be plotted by right-clicking the workspace and selecting ``Show Sample Shape``. This will plot all Sample, Container and Component shapes attached to the workspace, add a black arrow for the beam direction and colored arrows for the lattice vectors. For more details see :ref:`Mesh_Plots`.
+- Scripts generated by the Algorithm History Interface now include an import of ``mantid.simpleapi``.
+- Qt has been upgraded to version 5.15.
+- Normalisation (by maximum value) is available in Superplot
+- Drop support for ICAT v3
+- Mouse cursor now changes to a horizontal double arrow when hovering over peak and range markers in the fitting tool, indicating they can be dragged left and right.
+- In the error reporter, a label showing the maximum character limit of the additional context text has been added.
 
 
 Bugfixes
 --------
-.. amalgamate:: Workbench/Bugfixes
+- Fixed a bug that prevented arrays from displaying properly in sample logs
+- Fixed a bug where the user couldn't reselect a peak in the fitting tool after adding another
+- In the fitting widget, if a tie is edited to an invalid value Workbench will display an error and revert the tie to its previous value.
+- Fixed a bug where the superplot plot area shrunk on some data
+- Fixed a bug where data from workspaces (i.e the fit parameters) accessed from the Fit Browser could not be copied.
+- Saving script from history dialog in Linux will be saved with :code:`.py` extension
+- When a file with mantid algorithms is loaded, and the user accepts workbench adding the mantid API import line, the new import is always added to the top of the file
+- Fixed a bug where waterfall plots were updated unconditionally during ADS updates
+- The sensitivity of vertial and horizontal lines on plots, including the peak picker and range selector tools, has been increased.
+- Fixed bug where Load and Rebin showed up twice (upper and lower case) in the algorithms tab search.
+- Fixed a bug where property values were shortened in history
+- Fixed bug in the Mean algorithm where an invalid input workspace name would cause Mantid to crash.
+- Fixed a bug where the user could drag the width handles of an unselected peak in the fit browser.
+- Fix inconsistent text size when opening multiple script editor tabs on macOS
+- Fixed bug with AddSampleLogMultiple where including log types and keeping the parser box ticked would crash Mantid.
+- Fixed bug where Delta and Elastic function peaks were not connected to the fitting window GUI.
+- Fixed a bug where collapsing plots using a splitter handle could result in a crash.
+- Grid lines will not turn on unexpectedly when changing axis format in a 3-D plot.
+- Fixed bug where marker labels became unattached when zooming and panning.
+- When plotting with exported script, the "Show Minor Gridlines" checkbox in the Axes tab will be ticked and fixed toggle grid lines button affecting minor lines.
```

Comment:
I'm not sure what this sentence is trying to say. If it's not just unclear to me, then it might be good to clarify if we can?